src/modules/account/views/ViewAccountAssets.vue

```diff
@@ -109,6 +112,47 @@ export default {
                 return false
             }
         }
+        const publicAccount = PublicAccount.createFromPublicKey(acc.value?acc.value.publicKey:'0'.repeat(64),AppState.networkType)
+        const noBalanceMosaics = ref([])
+        const fetchNoBalanceMosaics = async()=>{
+            if(!acc.value){
+                return 
+            }
+            let assetAPI = AppState.chainAPI.assetAPI
+            let mosaicQueryParams = new MosaicQueryParams
+            mosaicQueryParams.ownerPubKey = publicAccount
+            let ownerAssets = await assetAPI.searchMosaics(mosaicQueryParams)
+            ownerAssets.mosaicsInfo.forEach(mosaic=>{
+                let mosaicId = new MosaicId([mosaic.mosaicId.id.lower,mosaic.mosaicId.id.higher])
+                let findAsset =  acc.value.assets.find(asset=>asset.idHex == mosaicId.id.toHex())
+                if(!findAsset){
+                    
+                    noBalanceMosaics.value.push({
+                        id: mosaicId.id.toHex(),
+                        balance: 0,
+                        isCreator: true
+                    });
+                }
+            })
+            let mosaicIds = []
+            noBalanceMosaics.value.forEach(mosaic=>{
+                mosaicIds.push(new MosaicId(mosaic.id))
+            })
+            let mosaicNames = await AppState.chainAPI.assetAPI.getMosaicsNames(mosaicIds)
+            noBalanceMosaics.value.forEach((mosaic,index)=>{
```

Comment:
wrong approach, please wait for new PR
- missing pagination and loading
- missing asset name, data loading in batch, limit number per request

Reply:
ok noted, thanks